build/npm.go

```diff
@@ -78,3 +92,13 @@ func (nm *NpmModule) AddArtifacts(artifacts ...entities.Artifact) error {
 	partial := &entities.Partial{ModuleId: nm.name, ModuleType: entities.Npm, Artifacts: artifacts}
 	return nm.containingBuild.SavePartialBuildInfo(partial)
 }
+
+// This function discard the npm command in npmArgs, and keep over only the command flags.
+// It is necessary for the npm command to come before the npm command flags in npmArgs in order to work the function correctly.
```

Comment:
The above description includes multiple grammar issuues.

Reply:
Done